src/components/Handbook/MainPage/Featured/styled.ts

```diff
@@ -0,0 +1,112 @@
+import styled, { media } from '@theme'
+import { Link } from 'gatsby'
+import Img from 'gatsby-image'
+
+import { Container, Title as BaseTitle, Text as BaseText } from '../../../UI'
+
+import ArrowIcon from '../../common/Arrow'
+
+export const Wrapper = styled(Container)`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 3rem;
+
+  margin-bottom: 5rem;
+
+  ${media.large} {
+    grid-template-columns: 1fr;
+  }
+`
+
+export const Content = styled.div`
+  max-width: 50%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  ${media.medium} {
+    max-width: 100%;
+  }
+
+  transition: opacity ${({ theme }) => theme.transitions.ease()};
```

Comment:
This rule is in the wrong spot, it would be better grouped with other ones above